src/main/java/bio/terra/dao/DatasetDao.java

```diff
@@ -187,31 +196,23 @@ private Dataset retrieveWorker(String sql, MapSqlParameterSource params) {
     }
 
     public MetadataEnumeration<DatasetSummary> retrieveDatasets(
-        int offset,
-        int limit,
-        String sort,
-        String direction,
-        String filter
-    ) {
+            int offset,
+            int limit,
+            String sort,
+            String direction,
+            String filter) {
```

Comment:
consider reverting?

Reply:
How come?

Reply:
Because it is at the wrong indentation level. Don't worry, Rori. Checkstyle will git 'im!

Reply:
This is something I'd like to discuss.

1. It's not something we enforce with checkstyle
2. I don't like having these parameters at the same indentation level as the code. I'd prefer to be able to visually tell where the params end and the code begins.
3. Whatever we decide, let's add it to checkstyle and to our editor styling configs.